modules/heat_conduction/include/kernels/ADHeatConduction.h

```diff
@@ -0,0 +1,34 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+#ifndef ADHEATCONDUCTION_H
+#define ADHEATCONDUCTION_H
+
+#include "ADKernel.h"
+#include "MaterialProperty.h"
```

Comment:
Ideally this would be a forward declaration but since we've obfuscated the `ADMaterialPropertyObject` class template, I think this is best.

Reply:
This isn't actually required, so I'll just remove it